includes/class-core.php

```diff
@@ -555,7 +555,7 @@ public function append_footer_script_for_collapsed_flexible_content_layouts() {
 		<script>
 			(function( $ ) {
 				$( document ).ready( function( ) {
-					$('.acf-flexible-content .layout:not([data-layout="text"]').addClass('-collapsed');
+					$('.acf-flexible-content .layout:not([data-layout="text"],.acf-clone)').addClass('-collapsed');
```

Comment:
Lurer på om kommaseparert `:not` har noen problemer i enkelte nettlesere. Bør trolig bruke `.layout:not([data-layout="text"]):not(.acf-clone)`